components/tests/ui/resources/robot.template

```diff
@@ -20,7 +20,7 @@ ${SERVER_ID}            1
 ${BROWSER}              Firefox
 ${DELAY}                0
 ${WAIT}                 50
-${TIMEOUT}              1
+${TIMEOUT}              60
```

Comment:
Timeout for all loops ``Wait until keyword succeed`` is meant to be 1 min. I accidentally put 1s rather then 1min=60sec.